sdk/storage/storage-blob/src/AppendBlobClient.ts

```diff
@@ -256,7 +254,7 @@ export class AppendBlobClient extends BlobClient {
   public async create(
     options: AppendBlobCreateOptions = {}
   ): Promise<Models.AppendBlobCreateResponse> {
-    const aborter = options.abortSignal || Aborter.none;
+    const aborter = options.abortSignal || AbortSignal.none;
```

Comment:
Isn't the abortSignal optional in the options passed to the `create` method below?
If so, then why do we set it to `AbortSignal.none`?
Can't the underlying code handle the case of no abort signal?

Reply:
I think you're right, I'm not sure why Jeremy added it in the first place.
I can remove it and see if that breaks anything. 
We might have to do the same at many other places.

Reply:
Created a new issue #4570 to track and update everywhere we do this in all the storage-packages